Estimate Precision Visualization.ipynb

```diff
@@ -190,7 +190,7 @@
    "source": [
     "def p_d(tx, dx, n, P_rx, P_tx, D_std):\n",
     "    modeledDistance = RSSItoDistance(P_rx, P_tx, n)\n",
-    "    return ss.norm.pdf(np.linalg.norm(dx - tx), loc=modeledDistance, scale=D_std)\n",
+    "    return math.exp(-(((np.linalg.norm(dx-tx)-modeledDistance)/D_std)**2)/2) / (math.sqrt(2*math.pi) * D_std)\n",
```

Comment:
Is there a reason that we are using math.exp, math.sqrt, and math.pi instead of their numpy equivalents?

I would say that we should add a Markdown cell above to write out the math a bit more explicitly, since it is a bit hard to follow as a one-liner.  You may want to try breaking apart the one-liner for readability, but only if it doesn't impact your runtime (it might not).

Reply:
I benchmarked math and the NumPy equivalents and found that math was faster for all operations and constants.
Did a bit of research online and found that math functions typically get replaced with an inline equivalent while the NumPy version remains as a function call, leading to the difference in runtime.

I also edited the Markdown cell right above to include the math. Let me know if I should change the format though.

Finally, I broke down the one-liner to calculate x separately.

Reply:
ok